src/code_8001F980.c

```diff
@@ -1618,61 +1626,57 @@ void func_80026A48(Player *player, s8 arg1) {
     }
 }
 
-#ifdef MIPS_TO_C
-//generated by m2c commit b52d92c2340f6f4ba1aafb464188bb698752fbb0 on Jul-30-2023
-typedef u8 wheels[4][128];
-
-extern wheels gKartMario168Wheel0;
-
-extern wheels *gKartMarioWheels0[];
+// Properly define struct pointers, see framebuffers.h comment for more information.
+#ifdef AVOID_UB
+#define D_802F1F80_WHEEL(a, b, c) &D_802F1F80[a][b][c].wheel_palette
```

Comment:
Shouldn't struct member names be `camelCase` not `snake_case`? I don't mind an exception here if there's a good reason for it though.

Reply:
No reason, just what I came up with when I originally added the struct.